app/src/common/shared/com/igalia/wolvic/browser/api/WFactory.java

```diff
@@ -19,7 +19,12 @@ public class WFactory {
     }
 
     public static @NonNull WSession createSession(@Nullable WSessionSettings settings) {
-        return new SessionImpl(settings);
+        return new SessionImpl(settings, null);
+    }
+
+    public static @NonNull WSession createSession(
+            @Nullable WSessionSettings settings, @NonNull Context context) {
+        return new SessionImpl(settings, context);
```

Comment:
note: The overload of createSession is added because current chromium implementation needs Activity context to initialize window.

Reply:
Removed.